tasks/request_from_archive/request_from_archive.py

```diff
@@ -167,51 +168,43 @@ def get_glacier_recovery_type(config: Dict[str, Any]) -> str:
     # Look for config override
     recovery_type = config.get(CONFIG_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY, None)
     if recovery_type is not None:
-        if recovery_type in VALID_RESTORE_TYPES:
+        LOGGER.info(
+            f"Using restore type of {recovery_type} "
+            f"found in the configuration {CONFIG_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY} key."
+        )
+    else:
+        # Look for default from TF
+        recovery_type = os.getenv(OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY, None)
+        if recovery_type is not None:
             LOGGER.info(
                 f"Using restore type of {recovery_type} "
-                f"found in the configuration {CONFIG_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY} key."
+                f"found in the environment {OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY} key."
             )
         else:
-            LOGGER.error(
-                f"Invalid restore type value of '{recovery_type}' "
-                f"found in the configuration {CONFIG_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY} key."
-            )
-            raise ValueError(
-                f"Invalid restore type value in configuration "
-                f"{CONFIG_DEFAULT_RECOVERY_TYPE_OVERRIDE_KEY} key."
+            recovery_type = "Standard"
```

Comment:
Default now set here. Since TF will always set the os.environ default, this did not affect customers.

Reply:
Instead of setting a value of `recovery_type = "Standard"` here I wonder if we shouldn't raise a `ValueError` here? In order to trigger this, that would mean, someone would have to purposefully delete the environment key from the lambda or set the TF value to None. I feel like we should throw an error and exit in that case instead of setting a default value of `Standard`. Thoughts?